### PR TITLE
aligns sepolia with mainnet parameters

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

**Comment by kaleb-keny** [comment]
@noisekit can we get this merged please, i'm signing off for today, I want to send this to the chain please for testing before we go live with mainnet, to make sure we are good on everything.

File: tomls/omnibus-base-sepolia-andromeda/perps/eth.toml [hunk 1]
```diff
@@ -54,4 +53,3 @@ defaultValue = "<%= MaxUint256 %>"
 defaultValue = "0.0005"
 
 [setting.perpsEthEndorsedLiquidator]
-defaultValue = "0x95A61Fa7454CA5f6A3CE01724e306Cd14a22D306"
```
[the remark this answers is not in the document]
Comment (kaleb-keny) [reply tomls/omnibus-base-sepolia-andromeda/perps/eth.toml]: thank you for pointing it out, fixed @0xjocke

File: tomls/omnibus-base-sepolia-andromeda/perps/rune.toml [hunk 2]
```diff
@@ -17,13 +17,13 @@ defaultValue = "0.0002"
 defaultValue = "0.0010"
 
 [setting.perpsRuneMaxMarketSize]
-defaultValue = "35600"
+defaultValue = "0"
```
Comment (noisekit) [inline tomls/omnibus-base-sepolia-andromeda/perps/rune.toml:20]: Zero for testnet?

Comment (kaleb-keny) [reply tomls/omnibus-base-sepolia-andromeda/perps/rune.toml:20]: interesting 🤔

**Review by noisekit — APPROVED** [review]

**Pull request merged**